test/Microsoft.SqlTools.ServiceLayer.UnitTests/Utility/SrTests.cs

```diff
@@ -15,10 +18,114 @@ public class SrTests
         [Fact]
         public void SrPropertiesTest()
         {
-            Assert.NotNull(ServiceLayer.SR.QueryServiceSubsetBatchNotCompleted);
-            Assert.NotNull(ServiceLayer.SR.QueryServiceFileWrapperWriteOnly);
-            Assert.NotNull(ServiceLayer.SR.QueryServiceFileWrapperNotInitialized);
-            Assert.NotNull(ServiceLayer.SR.QueryServiceColumnNull);
+            Assert.NotNull(SR.QueryServiceSubsetBatchNotCompleted);
```

Comment:
I'll add some unrelated tests to this PR to push code coverage metric positive.  This test hits a bunch of localization properties so that we don't have all that code counted as uncovered in the test runs.